docker-compose.yml

```diff
@@ -42,6 +42,8 @@ services:
     restart: always
     env_file:
       - .prod.env
+    environment:
+      - GEVENT_WORKER=0
```

Comment:
let's add also legacy option with 'NO_MONKEY_PATCH'

also I'd be explicit in gunicorn server as well